### PR TITLE
No dpkg root

### PR DESCRIPTION
**Pull request by JL2210**
Fixed need for root priveleges with dpkg. Dpkg does not need sudo anymore.

### CODE REVIEW

File: packages/dpkg.rb [hunk 1]
```diff
@@ -53,9 +40,13 @@ def self.install
     Dir.chdir ("#{CREW_DEST_PREFIX}/bin") do
       system 'mv dpkg dpkg-run'
       system "echo '#!/bin/bash' > dpkg"
-      system "echo 'dpkg-run --force-depends --root=#{CREW_PREFIX} \"$@\"' >> dpkg"
+      system "echo 'dpkg-run --force-not-root --force-depends --root=#{CREW_PREFIX} \"$@\"' >> dpkg"
       system 'chmod a+x dpkg'
     end
+    Dir.chdir ("#{CREW_DEST_PREFIX}") do
+      puts "This will make dpkg work without root privileges".lightred
+      system "sudo", "chown", "-R", "chronos:chronos", "./var/lib/dpkg"
```
Comment (uberhacker) [inline packages/dpkg.rb]: Instead of `chronos`, please abstract with `#{USER}` instead.  See `lib/const.rb`.

**Comment by uberhacker** [comment]
Please remove all the other packages from this PR.  Create a separate branch from your other working branch.

File: packages/dpkg.rb [hunk 2]
```diff
@@ -53,9 +40,13 @@ def self.install
     Dir.chdir ("#{CREW_DEST_PREFIX}/bin") do
       system 'mv dpkg dpkg-run'
       system "echo '#!/bin/bash' > dpkg"
-      system "echo 'dpkg-run --force-depends --root=#{CREW_PREFIX} \"$@\"' >> dpkg"
+      system "echo 'dpkg-run --force-not-root --force-depends --root=#{CREW_PREFIX} \"$@\"' >> dpkg"
       system 'chmod a+x dpkg'
     end
+    Dir.chdir ("#{CREW_DEST_PREFIX}") do
+      puts "This will make dpkg work without root privileges".lightred
```
[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/dpkg.rb]: You actually don't even need to output any message here.  A simple comment would work.

**Comment by uberhacker** [comment]
Not sure how you included the heroku package but that needs removed now.

**Comment by uberhacker** [comment]
I still see the heroku package.  I would suggest you just delete this branch and create a new PR on a clean branch.

**Comment by JL2210** [comment]
I removed the changes. Git keeps screwing up. I don't know why the changes from other branches are showing up here. I fixed it.

**Comment by JL2210** [comment]
@uberhacker, can you help me on this?

**Comment by uberhacker** [comment]
I think you need to rethink your workflow.  You should be merging changes from the Chromebrew master branch into your PR branch.  It should work like below:
```
$ cd /path/to/chromebrew
$ git remote add chromebrew git@github.com:skycocker/chromebrew.git
$ git checkout chromebrew/master
$ git pull chromebrew master
$ git checkout your-pr-branch
$ git merge master
$ git push origin your-pr-branch
```

**Comment by JL2210** [comment]
@uberhacker 
error: pathspec 'chromebrew' did not match any file(s) known to git.

**Comment by JL2210** [comment]
I've been using this as a reference:
(blog)[https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/]

**Pull request closed without merging**